### PR TITLE
Mention about an API incompatibility of Ruby 2.3.4

### PR DESCRIPTION
**Pull request by unak**
Please review my poor English.

### CODE REVIEW

File: en/news/_posts/2017-03-30-ruby-2-3-4-released.md [hunk 1]
```diff
@@ -17,6 +17,14 @@ And this release contains a bug fix of Symbol#hash to be non-deterministic.
 This is a regression on the 2.3 series before 2.3.4.
 See [Bug #13376](https://bugs.ruby-lang.org/issues/13376) for more details.
 
+## Known Problem (this section was added at 11 Apr 2017)
+
+An API incompatibility has been found with Ruby 2.3.4.
+It is the accidentally removal of an API named `rb_thread_fd_close`.
```
Comment (hlcfan) [inline en/news/_posts/2017-03-30-ruby-2-3-4-released.md]: accidental

File: en/news/_posts/2017-03-30-ruby-2-3-4-released.md [hunk 2]
```diff
@@ -17,6 +17,14 @@ And this release contains a bug fix of Symbol#hash to be non-deterministic.
...
+We will fix this problem at the next release, but if you are faced with
```
Comment (hlcfan) [inline en/news/_posts/2017-03-30-ruby-2-3-4-released.md]: facing the problem

File: en/news/_posts/2017-03-30-ruby-2-3-4-released.md [hunk 3]
```diff
@@ -17,6 +17,14 @@ And this release contains a bug fix of Symbol#hash to be non-deterministic.
...
+the problem now and have to overcome it immediately, use this patch:
```
Comment (hlcfan) [inline en/news/_posts/2017-03-30-ruby-2-3-4-released.md]: Maybe change `have` to `need`?

**Comment by unak** [comment]
Thank you, @hlcfan !

**Pull request merged**